### ISSUE TITLE
Improvements to waypoints

### ISSUE THREAD

**Issue by raydeejay** [GT3CH1/minecraft-mod#55]
**Is your feature request related to a problem? Please describe.**
Waypoints are managed through the settings menu. Additionally, they show in every dimension.

**Describe the solution you'd like**
I think waypoint management would make more sense in the main menu, since it's likely to be done often, while the rest of the settings seem more the kind of thing that's configured once and left alone for long.

As for showing in every dimension, asking for the dimension and then checking it before drawing the tracer/ESP should do the trick.

**Comment by GT3CH1** [comment]
I've implemented per-dimension checking for waypoints (something that should've been done long ago). 
As for showing the management/settings menu, I don't think adding it to the main menu makes a lot of sense, as that menu is used for toggling the mods and not managing them. However, I have pulled the waypoints into their own settings category, so that it is not nested under "render" in the settings menu. The toggle for the mod is still under the "Render" group.
